src/utils.rs

```diff
@@ -69,6 +69,25 @@ where
     Ok(i != 0)
 }
 
+pub fn de_from_u8_with_check<'de, D>(deserializer: D) -> Result<bool, D::Error>
```

Comment:
Why creating a new function? The function just above is doing exactly the same. You could improve the function just above?

Reply:
I created this PR to solve the problem.
https://github.com/patochectp/navitia_model/pull/3